nb_service/api/serializers.py

```diff
@@ -13,7 +13,11 @@ class Meta:
 from drf_yasg.utils import swagger_serializer_method
 from rest_framework import serializers
 from django.contrib.contenttypes.models import ContentType
-from netbox.api import ChoiceField, ContentTypeField, WritableNestedSerializer
+
+
+from netbox.api.fields import ChoiceField, ContentTypeField
```

Comment:
Since version 3.3 changes the namespace of the modules, we need to add a IF clause to keep backward compatibility.
So please add something like this.
```
from packaging import version

NETBOX_CURRENT_VERSION = version.parse(settings.VERSION)
if NETBOX_CURRENT_VERSION >= version.parse("3.3") :
    from netbox.api.fields import ChoiceField, ContentTypeField
    from netbox.api.serializers import WritableNestedSerializer
else:
    from netbox.api import ChoiceField, ContentTypeField, WritableNestedSerializer
```